incubator/azuremonitor-containers/templates/omsagent-daemonset.yaml

```diff
@@ -24,8 +24,12 @@ spec:
    serviceAccountName: omsagent
    {{- end }}
    containers:
-     - name: omsagent 
+     - name: omsagent
+       {{- if eq .Values.omsagent.domain "opinsights.azure.cn" }}
+       image: "dockerhub.azk8s.cn/microsoft/oms:{{ .Values.omsagent.image.tag }}"
+       {{ else }}
```

Comment:
so if a user specified an invalid domain, we default to public cloud ?

Reply:
yes, public cloud is the default one.